docs/source/index.rst

```diff
@@ -80,6 +80,7 @@ Expected output:
    testing
    extend
    deck
+   extras.accelerators
```

Comment:
this is because some of them are under types.extend and some under tasks.extend.

While accelerators is under extras, but it is not a type or a task extension. it is just a helper to help configure accelerators